src/components/content/ContentComponent.tsx

```diff
@@ -0,0 +1,40 @@
+import DetailBlogComponent from '../detail_blog/DetailBlogComponent';
+import ListBlogComponent from '../list-blog/ListBlogComponent';
+import './ContentStyle.css';
+
+const topicsBlog = [
+    {
+        name: 'Front-end',
+    },
+    {
+        name: 'Back-end',
+    },
+    {
+        name: 'Fullstack',
+    },
+
+]
+function ContentComponent() {
+    return (
+        <div>
+            <div className="row">
```

Comment:
Don't name common like this, you should use [BEM](http://getbem.com/introduction/). Ex: `content__row`